src/server/DeviceServer.js

```diff
@@ -361,9 +362,16 @@ class DeviceServer {
       );
 
       const description = await device.getDescription();
-      const { uuid } = FirmwareManager.getAppModule(
-        description.systemInformation,
-      );
+      // const { uuid } = FirmwareManager.getAppModule(
+      //   description.systemInformation,
+      // );
+      try {
+        uuid = FirmwareManager.getAppModule(
+          description.systemInformation,
+        )[0];
```

Comment:
Yeah -- `FirmwareManager.getAppModule` uses `find` which will return a single module.

The module returns `{uuid: string, .....otherStuff}`

when you do `const {uuid} = FirmWareManager.getAppModule` it uses destructuring so you only get the `uuid` prop out of the returned object.

I think the `platformID` check (above) will be fine but we probably don't need this change.